actors/builtin/miner/election.go

```diff
@@ -0,0 +1,31 @@
+package miner
+
+import (
+	"github.com/filecoin-project/specs-actors/actors/abi"
+	"github.com/filecoin-project/specs-actors/actors/builtin/reward"
+	"github.com/filecoin-project/specs-actors/actors/util/adt"
+)
+
+func MinerEligibleForElection(store adt.Store, mSt *State, rSt *reward.State, minerActorBalance abi.TokenAmount, currEpoch abi.ChainEpoch) (bool, error) {
```

Comment:
My impulse would have been to make this a method on miner state. Is there a reason why you didn't? I don't feel strongly about it, although I'll be a little unhappy about adding another file to the miner directory if it never gets more than one free function.

Reply:
It would be better to pass in rSt.ThisEpochReward as a parameter than to expose this method to all of reward state.

Reply:
I'm going off of @anorth's cue from the issue this closes:
> Add a single MinerEligibleForElection function that takes parameters as necessary (power state, miner state, current balance)

I think the idea is to make it easier for lotus so they don't have to do any state processing.

Regardless of how we structure we can move to another file.  Any suggestions where to put it?

Reply:
Lotus will need to do an equivalent amount of state processing whether it's a method on miner state or miner state is passed in as a parameter. I was thinking it should simply return to `miner_state.go` unless there's more anticipated election related logic.